src/bot/commands/get_filter.rs

```diff
@@ -12,8 +15,8 @@ pub struct GetFilter {
 }
 
 impl GetFilter {
-    pub fn run(&self) {
-        self.execute(&self.message);
+    pub fn run(&self, db_pool: Pool<ConnectionManager<PgConnection>>, api: Api, message: Message) {
```

Comment:
I think you solved conflicts in favor of older changes. please check the master branch, we don't need to pass api and db_pool to the commands

let's get rid of them here and all other commands